fedoracommunity/widgets/package/bugs.py

```diff
@@ -43,21 +43,29 @@ def prepare(self):
 
         releases = []
         self.filters = {'package': self.package}
-        pkgdb = get_connector('pkgdb')
-        collections = pkgdb.get_collection_table(active_only=True)
-
-        for id, collection in collections.items():
-            name = collection['name']
+        bodhi = get_connector('bodhi')
+        releases_obj = bodhi.call('releases', {})
+        total_pages = releases_obj['pages']
+        collections = releases_obj['releases']
+        if(total_pages > 1):
+            for i in range(2,total_pages+1):
+                temp = bodhi.call('releases?page='+str(i), {})['releases']
+                collections.extend(temp)
+
+        for collection in collections:
+            if collection['state'] == 'archived':
+                continue
+            name = collection['id_prefix']
             ver = collection['version']
-            label = "%s %s" % (name, ver)
+            label = collection['long_name']
             value = str(ver)
             if ver == 'devel':
```

Comment:
I don't think Bodhi has a version called `devel`. I think this if statement will need to be removed. Because of this, you'll have to add Rawhide after your loop to the `releases` variable.